DiscordChatExporter.Cli/Commands/Base/PartitionValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using ByteSizeLib;
+using CliFx;
+using CliFx.Extensibility;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace DiscordChatExporter.Cli.Commands.Base
+{
+    public class PartitionValidator : BindingValidator<string>
+    {
+        public override BindingValidationError? Validate(string value)
+        {
+            return int.TryParse(value, out _) ? Ok() :
+                ByteSize.TryParse(value, out _) ?
+                Ok() :
+                Error(@"The value must be numeric or represent a file size (e.g. 25mb).");
+        }
+    }
```

Comment:
I now realize it's not immediately obvious, but `BindingValidator` is triggered after the value is already converted, so this should be `BindingValidator<IPartitioner>` instead. However, seeing as the validation is in a way already handled in `BindingConverter`, I think this validator is not necessary at all.

Reply:
Created https://github.com/Tyrrrz/CliFx/issues/103